newrelic_plugin_agent/plugins/base.py

```diff
@@ -345,7 +345,7 @@ def http_get(self, url=None):
         :rtype: requests.models.Response
 
         """
-        LOGGER.debug('Polling %s Stats at %s',
+        LOGGER.info('Polling %s Stats at %s',
```

Comment:
This should not be merged. I think I did that as a separate commit for this reason.